### PR TITLE
bsunanda:Run2-alca32 Remove some of the magic numbers and update the calibration macro

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/IsolatedParticles/interface/CaloConstants.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  static const double etaBEHcal= 1.392; //Transition eta between HB and HE
+  static const double zBackHE  = 549.3; //Back of HE
+  static const double rBackHB  = 288.8; //Back of HB
+  static const double rFrontHO = 384.8; //Front of HO
+  static const double zFrontHF = 1115.; //Front of HF
+  static const double zFrontTE = 110.0; //Front of Tracker endcap
+  static const double zBackTE  = 290.0; //Back of Tracker endcap
+  static const double rBackTB  = 109.0; //Back of Tracker barrel
+  static const double etaBETrak= 1.705; //Transition eta between barrel-endcap of Tracker
+}
```
Comment (mmusich) [inline Calibration/IsolatedParticles/interface/CaloConstants.h]: @bsunanda isn't there any other way to get this numbers (e.g. from the Geometry?)

Comment (bsunanda) [inline Calibration/IsolatedParticles/interface/CaloConstants.h]: It is not very easy to do that. That is why I have put in those constants by hand
________________________________

**Comment by mmusich** [comment]
+1 
cleans the code of several classes in 
   * `Calibration/HcalCalibAlgos`
   * `Calibration/IsolatedParticles`
from magic numbers, but collecting them in a new namespace under `Calibration/IsolatedParticles/interface/CaloConstants.h`. Future improvement would be to get those numbers from the geometry instead

**Comment by davidlange6** [comment]
+1

**Pull request merged**